### PR TITLE
docs: update binding and validate docs 

### PR DESCRIPTION
**Pull request by Skyenought**
#### Check the PR title.

- [x] This PR title match the format: \<type\>(optional scope): \<description\>
- [x] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en: docs: update binding and validate docs 
zh(optional): 

#### (Optional) Which issue(s) this PR fixes:

https://github.com/cloudwego/hertz/issues/413

### CODE REVIEW

**Comment by li-jin-gou** [comment]
PTAL @chaoranz758

File: content/zh/docs/hertz/tutorials/basic-feature/binding-and-validate.md [hunk 1]
```diff
@@ -3,8 +3,8 @@ title: "绑定与校验"
 date: 2022-05-23
 weight: 8
 keywords: ["绑定与校验", "go-tagexpr", "tag", "参数绑定优先级"]
-description: "Hertz 支持的参数绑定与校验相关功能及用法。"
-
+description: >
+  Hertz 支持的参数绑定与校验相关功能及用法。
```
Comment (chaoranz758) [inline content/zh/docs/hertz/tutorials/basic-feature/binding-and-validate.md:7]: 这里为啥要改呢

File: content/zh/docs/hertz/tutorials/basic-feature/binding-and-validate.md [hunk 2]
```diff
@@ -64,29 +64,30 @@ func main() {
...
-| query | 绑定请求的 query 参数 |
-| cookie | 绑定请求的 cookie 参数 |
-| header | 绑定请求的 header 参数 |
-| json | 绑定请求的 body 内容 content-type -> `application/json`，绑定 json 参数 |
-| raw_body | 绑定请求的原始 body(bytes)，绑定的字段名不指定，也能绑定参数。（注：raw_body 绑定优先级最低，当指定多个 tag 时，一旦其他 tag 成功绑定参数，则不会绑定 body 内容。） |
-| vd | 参数校验，[校验语法](https://github.com/bytedance/go-tagexpr/tree/master/validator) |
-| default | 设置默认值 |
+| go tag   | 说明                                                                                                                                                                           |
+|:---------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| path     | 绑定 url 上的路径参数，相当于 hertz 路由`:param`或`*param`中拿到的参数。例如：如果定义的路由为：/v:version/example，可以把 path 的参数指定为路由参数：`path:"version"`，此时，url: http://127.0.0.1:8888/v1/example，可以绑定path参数"1" |
```
Comment (chaoranz758) [inline content/zh/docs/hertz/tutorials/basic-feature/binding-and-validate.md:69]: 代码块加空格

File: content/zh/docs/hertz/tutorials/basic-feature/binding-and-validate.md (hunk 1 above)
Comment (li-jin-gou) [reply content/zh/docs/hertz/tutorials/basic-feature/binding-and-validate.md:7]: 暂时不动，这里我看没有统一标准，如果后面有需要再统一吧 @Skyenought

File: content/zh/docs/hertz/tutorials/basic-feature/binding-and-validate.md [hunk 3]
```diff
@@ -3,8 +3,7 @@ title: "绑定与校验"
 date: 2022-05-23
 weight: 8
 keywords: ["绑定与校验", "go-tagexpr", "tag", "参数绑定优先级"]
-description: "Hertz 支持的参数绑定与校验相关功能及用法。"
```
Comment (chaoranz758) [inline content/zh/docs/hertz/tutorials/basic-feature/binding-and-validate.md:6]: 保持不变就行

File: content/en/docs/hertz/tutorials/basic-feature/binding-and-validate.md [hunk 4]
```diff
@@ -42,25 +39,42 @@ func main() {
 }
 ```
 
-## Supported tags and parameter binding priorities
+### APIs
+>
+> hertz version >= v0.7.0
+
+| API                   | 说明                                                                                                                                                                                                     |
```
Comment (chaoranz758) [inline content/en/docs/hertz/tutorials/basic-feature/binding-and-validate.md:46]: 英文文档里出现了中文

**Review by li-jin-gou — APPROVED** [review]

**Pull request merged**